gradle.properties

```diff
@@ -0,0 +1,2 @@
+vaadinVersion=24.6.3
+quarkusVersion=3.8.6
```

Comment:
Would be better to bump Quarkus to at least to the latest 3.12 version.
In the vaadin-quarkus main branch README, the pre-requisite is Quarkus 3.12+
BTW, should be updated also in the Maven version of the starter

Reply:
Good catch, I've upgraded it to 3.15.3 which is the latest LTS.

https://github.com/vaadin/base-starter-flow-quarkus/pull/191
https://github.com/vaadin/quarkus/pull/188